katdal/chunkstore.py

```diff
@@ -42,6 +42,10 @@ class StoreUnavailable(OSError, ChunkStoreError):
 class ChunkNotFound(KeyError, ChunkStoreError):
     """The store was accessible but a chunk with the given name was not found."""
 
+    def __str__(self):
+        """Avoid the implicit repr() of KeyError since we'll have explanatory text."""
+        return ChunkStoreError.__str__(self)
```

Comment:
I tried that and it didn't work, presumably because `ChunkStoreError` does not define its own `__str__`. The MRO then goes to `KeyError` before the base class.